project.godot

```diff
@@ -864,6 +864,8 @@ dialog_simple/clear_text_by_click_only=false
 dialog_simple/reading_speed_in_wpm=200
 dialog_simple/left_click_action="Speed up"
 dialog_simple/stop_talking_animation_on="End of audio"
+dialog_simple/max_time_to_disappear=1.0
```

Comment:
I don't recognize this setting and can't find it in the code (at least not in `develop`). Is this an old setting? Or am I just missing it somewhere?

Reply:
Hmm, that must be an old one. I'll remove it.